aldryn_events/forms.py

```diff
@@ -169,7 +169,9 @@ def clean(self):
         # to reverse, in case of success that would mean that the app_config
         # is correct and can be used.
         data = super(AppConfigPluginFormMixin, self).clean()
-        if not namespace_is_apphooked(data['app_config'].namespace):
+        app_config = data.get('app_config', None)
```

Comment:
is app_config a required field? If it is and is not in data then it means it failed validation.. I don't think we should raise the invalid namespace validation error because the real validation error was from app_config.